packages/rocketchat-webrtc/WebRTCClass.coffee

```diff
@@ -803,9 +803,11 @@ WebRTC = new class
 
 
 Meteor.startup ->
-	RocketChat.Notifications.onUser 'webrtc', (type, data) =>
-		if not data.room? then return
+	Tracker.autorun ->
```

Comment:
Nevermind, I see what you're saying now. Stop the autorun once the if block has run once.